package.json

```diff
@@ -170,7 +170,7 @@
     "@openzeppelin/contracts": "3.1.0",
     "@sentry/react": "^6.3.5",
     "@sentry/tracing": "^6.3.5",
-    "@truffle/contract": "^4.3.0",
```

Comment:
If you leave the 0 here will always fetch the latest version in yarn.lock

Reply:
it's not guaranteed to be the newest, someone could just rewrite the lockfile and the CI will build it with the version from lockfile because it installs with `--frozen-lockfile`

Reply:
web3 v.1.2.11 is still installed according to the lockfile

Reply:
but yeah if you do a clean install it installs the 4.3.18, but being a little bit explicit won't hurt imo